django_date_extensions/fields.py

```diff
@@ -93,9 +110,16 @@ def __len__(self):
 
 
 ansi_date_re = re.compile(r'^\d{4}-\d{1,2}-\d{1,2}$')
+prefix_date_re = re.compile(r'^([a-zA-Z]+[.,]?) (\d{4})$')
+prefix_date_reverse_re = re.compile(r'^(\d{4}) ([a-zA-Z]+[,.]?)$')
+
+try:
```

Comment:
In Django<1.9 the `to_python` is not called unless the custom field use meta class `models.SubfieldBase` hence we simply check for attribute error and fall back to only `models.CharField`